app/code/community/Businessfactory/Roihuntereasy/Model/Cron.php

```diff
@@ -24,16 +24,20 @@ public function scheduleFeed($schedule) {
      */
     public function createFeed($fileFormat)
     {
-        Mage::log($fileFormat . ": "  . __METHOD__ . " cron", null, 'cron.log');
+        if (!isset($fileFormat) || trim($fileFormat)==='' ) {
```

Comment:
In this case I recommend to do it like this:  `createFeed($fileFormat = "xml")` then you will know that default value is xml without magic inside the function, but you still have to resolve default value from jobConfig.